test/e2e/components/vl-map-layer.js

```diff
@@ -9,36 +9,8 @@ class VlMapLayer extends VlElement {
     return this.getAttribute('name');
   }
 
-  async getFeatures() {
-    return this.driver.executeScript(`return arguments[0].layer.getSource().getFeatures();`, this);
-  }
-
-  async getFeature(id) {
-    const feature = await this.driver.executeScript(`return arguments[0]._geoJSON.writeFeature(arguments[0].layer.getSource().getFeatures().find((feature) => feature.getId() === ${id}));`, this);
-    return JSON.parse(feature);
-  }
-
-  async isClustered() {
-    return this.hasAttribute('data-vl-cluster');
-  }
-
-  async getClusterDistance() {
-    return this.getAttribute('data-vl-cluster-distance');
-  }
-
-  async hasAutoExtent() {
-    return this.hasAttribute('data-vl-auto-extent');
-  }
-
-  async getAutoExtentMaxZoom() {
-    return this.getAttribute('data-vl-auto-extent-max-zoom');
-  }
-
-  async clickPointFeatureOnMap(id, map) {
-    const feature = await this.getFeature(id);
-    const {coordinates} = feature.geometry;
-    await map.scrollIntoView();
-    await map.clickOnCoordinates(coordinates);
+  static get TAG() {
```

Comment:
hoe komt het dat we deze capitalizen of is dit een afkorting van iets?

Reply:
Goh, het was iets constante-achtig maar het moest wel overridable zijn en by default een error throwen zodat het duidelijk is wanneer je het vergeet te overriden. Dus daarmee: een static getter die eigenlijk een constante voorstelt (daardoor de hoofdletters).
Wil het gerust ook wel lowercasen hoor, maakt me niet zoveel uit :)